wisdem/commonse/turbine_constraints.py

```diff
@@ -186,6 +186,99 @@ def compute(self, inputs, outputs, discrete_inputs, discrete_outputs):
         outputs["tip_deflection_ratio"] = delta * inputs["max_allowable_td_ratio"] / parked_margin
 
 
+class BladeRootSizing(om.ExplicitComponent):
```

Comment:
Why is this getting added to turbine_constraints?  Does it get inputs across different modules?  It looks like it might be better placed in RotorSE?

Reply:
Let me move it. I initially thought of having it here because of some future hub considerations. But these are not there right now, so I'm happy to move it

Reply:
That makes sense, but if you do anticipate having hub-related inputs it makes more sense now why you put it where you did.  Up to you depending on what you see for next steps